tests/browsercontext-route.spec.ts

```diff
@@ -212,3 +212,65 @@ it('should support the times parameter with route matching', async ({ context, p
   await page.goto(server.EMPTY_PAGE);
   expect(intercepted).toHaveLength(1);
 });
+
+it(`should overwrite post body`, async ({ context, server, page }) => {
+  server.setRoute('/load', (req, res) => {
+    res.write(`
+      <script>
+        (async () => {
+          await fetch('/empty.html', {
+            method: 'POST',
+            body: 'original',
+          });
+        })()
+      </script>
+    `);
+    res.end();
+  });
+
+  await context.route('**/empty.html', route => {
+    route.continue({
+      postData: 'replaced',
+    });
+  });
+
+  const [req] = await Promise.all([
+    server.waitForRequest('/empty.html'),
+    page.goto(server.PREFIX + '/load'),
+  ]);
+
+  const body = (await req.postBody).toString();
+  expect(body).not.toBe('original');
+  expect(body).toBe('replaced');
+});
+
+it(`should overwrite post body with empty string`, async ({ context, server, page, browserName }) => {
```

Comment:
```suggestion
it(`should overwrite post body with empty string`, async ({ context, server, page, browserName }) => {
  it.fail(browserName === 'firefox', `There's likely a FF-specific bug in Juggler or how we ser/de to Juggler.`);
```

FF is still broken here.